source/index.html

```diff
@@ -1,12 +1,168 @@
+        <li class="offers-list__item">
+          <a class="offers-list__link">Новые поступления</a>
+        </li>
+        <li class="offers-list__item">
+          <a class="offers-list__link">Распродажа</a>
+        </li>
+      </ul>
 
+      <p class="page-body__delivery">Бесплатная доставка по России</p>
+
+    </header>
+    <section class="promo">
+      <h1 class="title promo__title">Милые штучки ручной работы</h1>
+
+      <a class="promo__link"></a>
+      <a class="promo__link"></a>
+
+    </section>
+    <section class="product-week">
+      <h2 class="product-week__title">Вязаные корзинки</h2>
+      <p class="product-week__label">Товар недели</p>
+
+      <p class="product-week__desc">
+        Экологически чистая пряжа,
+        ручная работа. Доступны в шести расцветках, и трех размерах.
+      </p>
+
+      <table class="product-week__feature">
+        <tr>
+          <td class="product-week__feature__item">
+            Цвет:
+          </td>
+          <td class="product-week__feature__item">
+            "Тиффани"/серый
+          </td>
+        </tr>
+        <tr>
+          <td class="product-week__feature__item">
+            Диаметр:
+          </td>
+          <td class="product-week__feature__item">
+            25 см
+          </td>
+        </tr>
+        <tr>
+          <td class="product-week__feature__item">
+            Высота:
+          </td>
+          <td class="product-week__feature__item">
+            15 см
+          </td>
+        </tr>
+      </table>
+
+      <img class="product-week__img">
+
+      <p class="product-week__price"></p>
+      <a class="button">Заказать</a>
+
+    </section>
+    <section class="about-us">
+      <h2 class="title">Коротко о нас:</h2>
+
+      <ul class="about-us__list">
+        <li class="about-us__list__item">
+          Экологически чистые материалы
+        </li>
+        <li class="about-us__list__item">
+          Скандинавский стиль по российской цене
+        </li>
+        <li class="about-us__list__item">
+          Увеличивает лайкина фотографиях
+        </li>
+        <li class="about-us__list__item">
+          Связано вручную с любовью и умилением
+        </li>
+        <li class="about-us__list__item">
+          Поддержка отечественного производителя
+        </li>
+        <li class="about-us__list__item">
+          Поставляется в подарочной упаковке
+        </li>
+      </ul>
+
+    </section>
+    <section class="reviews">
+      <h2 class="title">Отзывы</h2>
+
+      <div class="review-body">
+        <p class="review-body__text">
+          Я молодая мама, но всё равно фотографии не набирали и близко такого количества лайков, как
+          у популярных инстамамочек.
+          В отчаянии, я накупила аксессуаров и игрушек в Мишке,
+          и мои фотографии сразу стали более стильными, а также набирают больше лайков!
+        </p>
+
+        <p class="review-body__autor"><span>Анастасия Красильникова</span> @misssssiskras</p>
```

Comment:
Знаете, а здесь скорее всего наоборот нужно в `span` обернуть логин пользователя, а не имя:
```suggestion
        <p class="review-body__autor">Анастасия Красильникова<span>@misssssiskras</span></p>
```